bundles/CoreBundle/src/EventListener/Frontend/RoutingListener.php

```diff
@@ -88,13 +84,6 @@ public function onKernelRequest(RequestEvent $event): void
         // resolve current site from request
         $this->resolveSite($request, $path);
 
-        // check for override redirects
-        $response = $this->redirectHandler->checkForRedirect($request, true);
-        if ($response) {
-            $event->setResponse($response);
-
-            return;
```

Comment:
the control is not returned anymore that means it get executed even RedirectResponse exists from SeoBundle\RoutingListener...right?

Reply:
Yes, you are right. But IMHO the `$event->setResponse($response);` will not be overwritten afterwards so it makes no different.
wdyt?